src/pybind/mgr/dashboard/services/ganesha.py

```diff
@@ -69,6 +66,18 @@ def _get_clusters_locations(cls):
                 else:
                     pool, namespace = pool_nm.split('/', 1)
 
+            # Check pool/namespace collision.
+            for clusters in [orch_result, result]:
+                for cluster_name, cluster_data in clusters.items():
+                    if cluster_data['pool'] == pool and cluster_data['namespace'] == namespace:
+                        orch_error_msg = ' that is deployed by the Orchestrator' \
+                            if cluster_data['type'] == ClusterType.ORCHESTRATOR else ''
+                        raise NFSException(
+                            f'Pool `{pool}` and namespace `{namespace}` are already in use by '
+                            f'NFS-Ganesha cluster called `{cluster_name}`{orch_error_msg}. '
```

Comment:
I'd put the above string inline as well (something I like about f-strings is that you can really read the final message, so I'd put everything there:
```suggestion
                            f'NFS-Ganesha cluster called `{cluster_name}`{" that is deployed by the Orchestrator"  if cluster_data['type'] == ClusterType.ORCHESTRATOR else ''}. '
```

Reply:
Done.